### ISSUE TITLE
Fatal error on latest version (0.410.0) on Windows

### ISSUE THREAD

**Issue by miltonbecker** [prisma/studio#725]
This happens only on the latest studio version - 0.410.0.

When I click on a project I get this:

![image](https://user-images.githubusercontent.com/5862165/124990151-5d0d7f00-e016-11eb-80b3-10c463d2f89f.png)

The error details are:

```
Error starting Prisma Client: {
  "message": "\n    Error: spawn typegraphql-prisma ENOENT\n        at notFoundError (C:/Users/myuser/AppData/Local/Programs/@prismastudio-electron/resources/app.asar/node_modules/cross-spawn/lib/enoent.js:6:26)\n        at verifyENOENT (C:/Users/myuser/AppData/Local/Programs/@prismastudio-electron/resources/app.asar/node_modules/cross-spawn/lib/enoent.js:40:16)\n        at ChildProcess.cp.emit (C:/Users/myuser/AppData/Local/Programs/@prismastudio-electron/resources/app.asar/node_modules/cross-spawn/lib/enoent.js:27:25)",
  "stack": "AggregateError: \n    Error: spawn typegraphql-prisma ENOENT\n        at notFoundError (C:/Users/myuser/AppData/Local/Programs/@prismastudio-electron/resources/app.asar/node_modules/cross-spawn/lib/enoent.js:6:26)\n        at verifyENOENT (C:/Users/myuser/AppData/Local/Programs/@prismastudio-electron/resources/app.asar/node_modules/cross-spawn/lib/enoent.js:40:16)\n        at ChildProcess.cp.emit (C:/Users/myuser/AppData/Local/Programs/@prismastudio-electron/resources/app.asar/node_modules/cross-spawn/lib/enoent.js:27:25)\n    at next (C:\\Users\\myuser\\AppData\\Local\\Programs\\@prismastudio-electron\\resources\\app.asar\\node_modules\\p-map\\index.js:43:14)\n    at C:\\Users\\myuser\\AppData\\Local\\Programs\\@prismastudio-electron\\resources\\app.asar\\node_modules\\p-map\\index.js:59:6\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)"
}
Error: Error starting Prisma Client: {
  "message": "\n    Error: spawn typegraphql-prisma ENOENT\n        at notFoundError (C:/Users/myuser/AppData/Local/Programs/@prismastudio-electron/resources/app.asar/node_modules/cross-spawn/lib/enoent.js:6:26)\n        at verifyENOENT (C:/Users/myuser/AppData/Local/Programs/@prismastudio-electron/resources/app.asar/node_modules/cross-spawn/lib/enoent.js:40:16)\n        at ChildProcess.cp.emit (C:/Users/myuser/AppData/Local/Programs/@prismastudio-electron/resources/app.asar/node_modules/cross-spawn/lib/enoent.js:27:25)",
  "stack": "AggregateError: \n    Error: spawn typegraphql-prisma ENOENT\n        at notFoundError (C:/Users/myuser/AppData/Local/Programs/@prismastudio-electron/resources/app.asar/node_modules/cross-spawn/lib/enoent.js:6:26)\n        at verifyENOENT (C:/Users/myuser/AppData/Local/Programs/@prismastudio-electron/resources/app.asar/node_modules/cross-spawn/lib/enoent.js:40:16)\n        at ChildProcess.cp.emit (C:/Users/myuser/AppData/Local/Programs/@prismastudio-electron/resources/app.asar/node_modules/cross-spawn/lib/enoent.js:27:25)\n    at next (C:\\Users\\myuser\\AppData\\Local\\Programs\\@prismastudio-electron\\resources\\app.asar\\node_modules\\p-map\\index.js:43:14)\n    at C:\\Users\\myuser\\AppData\\Local\\Programs\\@prismastudio-electron\\resources\\app.asar\\node_modules\\p-map\\index.js:59:6\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)"
}
    at ki.init (file:///C:/Users/myuser/AppData/Local/Programs/@prismastudio-electron/resources/app.asar/renderer/assets/index.js:1:103816)
```

When I dowgrade to version 0.394.0 it works again.

**Comment by janpio** [comment]
What does your `schema.prisma` file look like? Seems the customer generator `typegraphql-prisma` is causing a problem here somehow.

**Comment by miltonbecker** [comment]
Not sure that's a good idea - it's a production app and there are a few hidden fields.
Anything specific you want to look at?
There are a few enums, several models and quite a lot of relations.

The beginning is like this:
```
generator client {
  provider = "prisma-client-js"
}

generator typegraphql {
  provider = "typegraphql-prisma"
}

datasource db {
  provider = "postgresql"
  url      = env("DB_URL")
}
```

**Comment by janpio** [comment]
That might already be enough - thanks.

**Comment by miltonbecker** [comment]
Oh this might also be important: I'm running the studio on Windows but the `schema.prisma` is on the WSL - so the filepath to the schema starts with `\\wsl$\Ubuntu-20.04`.

**Comment by madebysid** [comment]
I'll check this out soon and let you know what I find

**Comment by madebysid** [comment]
Hey @miltonbecker this was likely happening on the Prisma CLI as well, right?

I've just pushed out a fix for the CLI that should fix this. You'll need to install a `dev` version of the CLI to check this out. (`npm i prisma@dev @prisma/client@dev`).

Do note that the `dev` versions are mostly meant for internal use, so I wouldn't recommend using it in a production app. This fix will also land on 2.28.0 (scheduled for July 27th).

I've also just released a new version of the Electron app, and this should be fixed on there as well. Let me know if you still see issues!
